catalyst-gateway/Earthfile

```diff
@@ -139,4 +139,24 @@ test:
         --service event-db-is-running
         RUN --mount=$EARTHLY_RUST_CARGO_HOME_CACHE --mount=$EARTHLY_RUST_TARGET_CACHE \
             cargo nextest run --release --run-ignored=only signed_docs
-    END 
\ No newline at end of file
+    END
+
+test-scylla:
+    FROM +builder-src
+
+    # setup scylla
+    RUN apt-get update && \
+        apt-get install -y --no-install-recommends \
+        wget gnupg ca-certificates && \
+        mkdir -p /etc/apt/keyrings && \
+        gpg --homedir /tmp --no-default-keyring --keyring /etc/apt/keyrings/scylladb.gpg \
+            --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys a43e06657bac99e3 && \
+        wget -O /etc/apt/sources.list.d/scylla.list http://downloads.scylladb.com/deb/debian/scylla-6.2.list && \
+        apt-get update && \
+        apt-get install -y --no-install-recommends \
+        scylla
+
+    # run the test
+    RUN --mount=$EARTHLY_RUST_CARGO_HOME_CACHE --mount=$EARTHLY_RUST_TARGET_CACHE \
+        scylla --options-file /etc/scylla/scylla.yaml --smp=2 --memory=4G --overprovisioned --developer-mode=1 & \
```

Comment:
I think you should copy the scylla.yaml inside the container before passing it as parameter?

Reply:
This file is from Scylla itself after newly installing scylla. Or should I copy from the existing file instead?